src/openapi/streaming/serializer/serializer-protobuf.js

```diff
@@ -0,0 +1,125 @@
+import * as baseProtos from './schemas/proto-base';
+import protobuf from 'protobufjs/dist/protobuf';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import base64 from '@protobufjs/base64';
+import log from '../../../log';
+
+const LOG_AREA = 'SerializerProtobuf';
+
+/**
+ * Protobuf Serialization
+ * @constructor
+ */
+function SerializerProtobuf() {
```

Comment:
Should I create interface (base serializer) for serializers? It would state clear contract for all serializers.